runtime/manta/src/lib.rs

```diff
@@ -121,10 +120,10 @@ pub fn native_version() -> NativeVersion {
 
 /// We assume that ~5% of the block weight is consumed by `on_initialize` handlers. This is
 /// used to limit the maximal weight of a single extrinsic.
```

Comment:
Comment needs to be updated for new code.

Reply:
This is deliberate. The comments are from parity. Here we use a more conservative setting.